script.js

```diff
@@ -0,0 +1,56 @@
+let collection = [];
+let arr = [];
+
+const inputTitle = document.getElementById('title');
+const inputAuthor = document.getElementById('author');
+const submitBtn = document.querySelector('.add-btn');
+const bookSection = document.querySelector('.books');
+
+const book = {
+  title: '',
+  author: '',
+};
```

Comment:
- [OPTIONAL] It will be nice to use unique `id` property for each book, since this will make it easier to remove books with duplicate values.